pack/tcu/uad.json

```diff
@@ -17,6 +17,42 @@
 		"type_code": "event",
 		"xp": 0
 	},
+	{
+		"code": "05230",
+		"cost": 3,
+		"deck_limit": 2,
+		"faction_code": "guardian",
+		"illustrator": "Tiziano Baracchi",
+		"name": "Telescopic Sight",
+		"pack_code": "uad",
+		"position": 230,
+		"quantity": 2,
+		"skill_agility": 1,
+    "skill_combat": 1,
+    "skill_intellect": 1,
+		"text": "Fast. Attach to a [[Firearm]] asset you control that takes up two hand slots. Attached asset cannot be used to attack enemies engaged with you.\n[reaction] When you perform a <b>Fight</b> action using attached asset, if you are engaged with no enemies, exhaust Telecopic Sight: This attack can target a non-[[Elite]] enemy at a connecting location. Ignore the aloof and retaliate keywords for this attack.",
+		"traits": "Item. Upgrade.",
+		"type_code": "event",
+		"xp": 3
+	},
+	{
+		"code": "05231",
+		"cost": 0,
+		"deck_limit": 2,
+		"faction_code": "seeker",
+		"flavor": "... and powers corrupts.",
+		"illustrator": "Michal Milkowski",
+		"name": "Knowledge is Power",
+		"pack_code": "uad",
+		"position": 231,
+		"quantity": 2,
+    "skill_intellect": 1,
+		"skill_willpower": 1,
+		"text": "Fast. Play only during your turn.\nChoose a [[Tome]] or [[Spell]] asset you control, or reveal a [[Tome]] or [[Spell]] asset from your hand. Resolve an [action] or [free] ability on that asset, ignoring all costs <b>(including its [action] cost if any)</b>. Then, if that asset was in your hand, you may discard it to draw 1 card.",
```

Comment:
The parenthetical here should use an `<i>` tag instead of a `<b>`

Reply:
Fixed!